bears/ruby/RubySyntaxBear.py

```diff
@@ -14,7 +16,8 @@ class RubySyntaxBear:
     Checks the code with ``ruby -wc`` on each file separately.
     """
     LANGUAGES = {'Ruby'}
-    REQUIREMENTS = {GemRequirement('ruby', '2.1.5')}
+    REQUIREMENTS = {GemRequirement('ruby', '2.1.5'),
```

Comment:
You're asking me ... ?  You added it :P da629530

But yes, I think this `GemRequirement` needs to be revisited, and I'd be happy to remove it as redundant, but maybe you had some reason why the `2.1.5` is there...?